content/help/addons/hero-image.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Hero Image
+tags:
+- Addons
+- Hero Image
+- Image upload
+category: help
+menu:
+  help:
+    parent: addons
+    identifier: addon-hero-image
+versioning:
+  added: 2.6
+---
+
+## Overview
+
+{{% cloudfeature %}}
+
+This plugin allows you to:
```

Comment:
Add a simpler description before you jump into inheriting from parent categories image. You also need to be very explicit here that this requires custom work to see it. By just reading the title, it sounds like you can plop this into any theme and it'll work without any work.